config/strings.yaml

```diff
@@ -80,3 +80,7 @@ error:
     heading: Uh oh, that's a 500.
     message: Perhaps try to search for something else instead?
 
+# warning messages
+warning:
+  duplicate: !!js/function (names) => `Multiple resources in this folder share the same name${':' + ' ' + names}. Only one will be accesible through Library.`
```

Comment:
That's what I originally had, but I was getting this error:
```
[0]   throw generateError(state, message);
[0]   ^
[0] YAMLException: cannot resolve a node with !<tag:yaml.org,2002:js/function> explicit tag at line 85, column 93:
[0]      ...  this folder share the same name: ${names}. Only one will be acc ...
```
Tried googling around a bit but I didn't find anything useful. What I have here is the only variation I tried that didn't throw the error 🙁

Reply:
Ah, I see. It's getting upset about a `:` character as part of a key not enclosed in quotes. Two other options I found:
```
... this folder share the same name&#58; ${names}...
```
where the character sequence doesn't trip up the parser

or
```
... this folder share the same name${":"} ${names}...
```
which might be only slightly better because it's more obvious the problem with the `:` character.

But I could go either way, it's unfortunate that the parser doesn't interpret the backticks as quotes the same way.

What do you want to do?

Reply:
html entity is the best option I think